.github/workflows/system-tests.yml

```diff
@@ -77,36 +47,27 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 'DataDog/system-tests'
-
       - name: Checkout dd-trace-js
         uses: actions/checkout@v4
         with:
           path: 'binaries/dd-trace-js'
-
       - name: Build weblog
         run: ./build.sh -i weblog
```

Comment:
the weblog only need to be built once, before all the scenario are executed, and can be used on the different scenarios. Not sure how to make that in github CI tbh

Reply:
We'd have to store everything in a tarball in some kind of setup job, upload that as an artifact, download it in dependent jobs, unpack it, then proceed in those jobs.

It wouldn't save us much time, and in fact might cost more time, since all jobs would have to wait for that one to complete. Yes, we're performing the same operation a ton of times, but they all happen in parallel, so timing-wise it doesn't matter much. It does save us on execution time in aggregate, but I'm not sure it's worth it at this point.